### PR TITLE
Releases/v3.0.0

### PR DESCRIPTION
**Pull request by andrewjl-mux**
## Improvements

- Repackage SDK as source distribution
- Add Swift Package Manager support
- Raise minimum deployment targets to iOS 13 and tvOS 13

## Known Issues

- Cocoapod pod spec linting fails on Xcode 14.3 and above due to Cocoapods/Cocoapods issue #11839. As a workaround use xcode-select to switch to Xcode 14.2 before linting.

### CODE REVIEW

File: Sources/MUXSDKStatsKaltura/MUXSDKPlayerBinding.swift [hunk 1]
```diff
@@ -6,16 +6,18 @@
...
 import MuxCore
-import AVFoundation
+import PlayKit
 
 @objc
 public class MUXSDKPlayerBinding: NSObject {
     private let MUXSDKPluginName = "apple-kaltura-mux"
-    private let MUXSDKPluginVersion = Bundle.main.infoDictionary?["CFBundleVersion"] as? String
-    
+    private let MUXSDKPluginVersion = "3.0.0"
```
Comment (daytime-em) [inline Sources/MUXSDKStatsKaltura/MUXSDKPlayerBinding.swift:19]: not a big deal, but just curious why this is preferable

**Review by daytime-em — APPROVED** [review]

File: Sources/MUXSDKStatsKaltura/MUXSDKPlayerBinding.swift (hunk 1 above)
Comment (andrewjl-mux) [reply Sources/MUXSDKStatsKaltura/MUXSDKPlayerBinding.swift:19]: Great question! 

`Bundle.main.infoDictionary?["CFBundleVersion"]` no longer returns the SDK version with certainty. 

When installing the SDK as a Swift Package, this API will return the application `CFBundleVersion` instead. With Cocoapods it's a bit more complicated but there's not guarantee `CFBundleVersion` matches the pod spec version. Instead it depends on the structure that Cocoapods will setup, which varies based on the spec, the Podfile, and the Cocoapods version.

Comment (andrewjl-mux) [reply Sources/MUXSDKStatsKaltura/MUXSDKPlayerBinding.swift:19]: This is because there's no SDK binary and by extension no `Info.plist` file.

Comment (daytime-em) [reply Sources/MUXSDKStatsKaltura/MUXSDKPlayerBinding.swift:19]: Thanks for the in-depth answer! 🥇

Comment (andrewjl-mux) [reply Sources/MUXSDKStatsKaltura/MUXSDKPlayerBinding.swift:19]: I do have some automation in mind for verifying this constant is what it should be, similar to how we verify the pod spec version using a script.

**Comment by andrewjl-mux** [comment]
Somethings up with BuildKite still

**Pull request merged**